relay-server/src/envelope.rs

```diff
@@ -791,6 +798,14 @@ impl Envelope {
         })
     }
 
+    pub fn get_trace_context(&self) -> Option<&TraceContext> {
```

Comment:
Getters in Rust do not have a `get_` prefix as per convention.

```suggestion
    pub fn trace_context(&self) -> Option<&TraceContext> {
```